### DISCUSSION TITLE
Is the Smart Chat feature now only available to supporters?

### DISCUSSION THREAD

**Discussion by morgen1990** [brianpetro/obsidian-smart-connections#681] [General]
I've spent a long time trying to troubleshoot this bcz the docs are written as if it is available to all--not just supporters. But no matter what I do, I am asked to log in with my supporter account whenever I try to use the Smart Chat feature. Is it only available to paying supporters now?

**Comment by brianpetro** [comment]
_The Smart Chat is available to everyone, it is **not** supporter-only!_

You must be mistaking API keys for cloud models with the supporter program.

To make Smart Chat more accessible to new users, I contributed a "community API key" that allows new users to use Smart Chat immediately without an API key.

After a few uses, a notification would pop up _after every chat message_, letting the user know they are using a shared API key for the community and should add their API key.

![Screenshot 2024-07-03 at 8 53 17 AM](https://github.com/brianpetro/obsidian-smart-connections/assets/1886014/7e0bc8cb-d26f-4556-921a-c7f7a24924ea)

Unsurprisingly, people were abusing the generosity, so the community API key was never available (the limit was used up for the day) anytime I checked.

So yesterday, I restricted the individual use of the community API key to a maximum of 20 uses. This should give new users sufficient time to test Smart Connections while preventing abuse.

![Screenshot 2024-07-03 at 8 58 50 AM](https://github.com/brianpetro/obsidian-smart-connections/assets/1886014/9b2d7c89-4002-4904-be93-856799af94d7)

Did you add your own API key?

🌴

**Reply by morgen1990** [reply]
Thanks for your quick reply--the problem was that for some reason the command " Open: Smart Chat GPT" was opening a different interface for interacting with the chatgpt api, not the "Smart Connections" interface shown in your screenshot. I figured it out. Thank you.
